engine/profiler/src/profiler_proc_utils.cpp

```diff
@@ -184,7 +184,7 @@ uint64_t dmProfilerExt::GetProcMemoryUsage()
     }
     fclose(fp);
 
-    long page_size = sysconf( _SC_PAGESIZE);
+    long page_size = sysconf(_SC_PAGESIZE);
```

Comment:
I just removed an extra space (it is easier to search for this function if it's without space)